CHANGELOG

```diff
@@ -2,6 +2,8 @@ Qtile x.xx.x, released XXXX-XX-XX:
     !!! Config breakage !!!
         - lazy.qtile.display_kb() no longer receives any arguments. If you passed it any arguments
           (which were ignored previously), remove them.
+        - The format argument for widget.Net() takes `down` and `down_suffix` instead of just `down`.
+          This facilitates formatting of `down` (the same for `up`).
```

Comment:
I think we need to add a message here that the precision now needs to be stated explicitly in `format`. People who have set their own `format` string just using `{up}` and `{down}` are going to see multiple decimal places appear where they didn't have them before.

I know no one is really going to bother reading the changelog but, when we get the inevitable bug reports, we can point them to this!

Reply:
Done!